docs/pbc/all/dynamic-multistore/202307.0/base-shop/install-and-upgrade/install-features/install-the-dynamic-multistore-cart-feature.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Install the Dynamic Multistore & Cart feature
```

Comment:
Why is `&` used as a connector? In our other connector IGs, we are using `+`. For example, [Install the Cart + Product Bundles feature](https://docs.spryker.com/docs/pbc/all/cart-and-checkout/202307.0/base-shop/install-and-upgrade/install-features/install-the-cart-product-bundles-feature.html)

Reply:
no specific reason. Didn't know that we use + for that